### PR TITLE
Hide truediv from swig 3.0.11

### PR DESCRIPTION
**Pull request by jschueller**
It is shadowed since https://github.com/swig/swig/commit/1079ba7

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -77,6 +78,8 @@ script:
     -DNLOPT_LIBRARY=~/.local/lib/libnlopt.so -DNLOPT_INCLUDE_DIR=~/.local/include
     .
   - make install -j4 && ctest -R pyinstallcheck -j4 --output-on-failure --timeout 100
+  - cmake -DUSE_SPHINX=OFF -DPYTHON_EXECUTABLE=/usr/bin/python3 .
+  - make install -j4 && ctest -R pyinstallcheck -j4 --output-on-failure --timeout 100
```
Comment (sofianehaddad) [inline .travis.yml]: care
sometimes you should also fix both PYTHON_INCLUDE_DIR and PYTHON_LIBRARY

Comment (jschueller) [inline .travis.yml]: yes, thanks

**Comment by jschueller** [comment]
@sofianehaddad unfortunately travis time is too tight to build py3k too

**Comment by sofianehaddad** [comment]
+1

**Pull request merged**